ui/warlock/inputs.ts

```diff
@@ -55,6 +55,12 @@ export const PetInput = InputHelpers.makeSpecOptionsEnumIconInput<Spec.SpecWarlo
 	changeEmitter: (player: Player<Spec.SpecWarlock>) => player.changeEmitter,
 });
 
+export const NewDPBehaviour = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecWarlock>({
+	fieldName: 'newDPBehaviour',
+	label: 'New DP behaviour',
```

Comment:
```suggestion
export const NewDPBehavior = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecWarlock>({
	fieldName: 'newDPBehavior',
	label: 'New DP behavior',
```

supernit: American English spelling "behavior" vs. UK English spelling "behaviour"